relay-server/src/utils/thread_pool.rs

```diff
@@ -116,6 +137,30 @@ impl WorkerGroup {
     }
 }
 
+#[cfg(unix)]
+fn set_current_thread_priority(kind: ThreadKind) {
+    let prio = match kind {
+        // The default priority needs no change.
+        ThreadKind::Default => return,
+        // Set a priority of `10` for worker threads,
+        // it's just important that this is a lower priority than default.
+        ThreadKind::Worker => 10,
```

Comment:
What is the default/how do you know 10 is less than the default?

Reply:
The default is 0, I'll add that as a comment 👍 .